### ISSUE TITLE
Some questions

### ISSUE THREAD

**Issue by sebakerckhof** [meteorhacks/cluster#31]
Hello,

I like the idea of mini-services using cluster. It makes my application more modular and as an added benefit I can scale out DDP traffic.
Now, I actually made a separate build process using gulp to separate my front-end from my back-end (as we have some specific requirements on how to serve the front-end).
Therefore I do not need webapp (since I do not want autoupdate etc either). 
Clusters package.js specifies it needs webapp. Therefore my first question is how hard this dependency is between cluster and webapp.

My second question is how session data is handled. If I make mini services using cluster and use the accounts package is session and authentication data shared between these services?

Also,

**Comment by arunoda** [comment]
It's hard to remove the webapp package since that's place where all the HTTP goes.
User authentication syncing is in the pipeline. It'll be available soon.
see: #14

**Comment by sebakerckhof** [comment]
Thanks for the fast reply! 
Well my situation is somewhat specific, but to summarize:
-I have control of the browser environment of my users, and everyone has websocket support
-I must serve my application from a different server than the one running meteor.

So in this case I'd actually never need HTTP going to the meteor server except for the websocket handshake, or does this also pass through webapp?

**Comment by arunoda** [comment]
Yes. it's also handled by webapp.

**Comment by sebakerckhof** [comment]
Thanks! In that case I'll just create a stripped down version of webapp, with static file serving etc thrown out.

**Comment by arunoda** [comment]
Okay.

**Comment by dropfen** [comment]
Hey @sebakerckhof, it would be cool, if you could open source it, and describe what the benefits are.

**Comment by sebakerckhof** [comment]
@dropfen yeah, I've seen quite a bunch of questions on the meteor forum from people that want to separate the client-side from server-side. I'm sure I'll do a write up and probably create an atmosphere package if it makes sense. Most of the build process stuff is in place, but I'm still figuring out what's the best way to keep the development fast (so you don't have to do a build on every change).
